### DISCUSSION TITLE
Template simplification

### DISCUSSION THREAD

**Discussion by taypo** [tschuehly/spring-view-component#21] [Ideas]
If we place the views with the same package as controllers, do we really need this line:

```
<!--/*@thymesVar id="d" type="de.tschuehly.example.thymeleafjava.web.simple.SimpleViewComponent.SimpleView"*/-->
```

I feel omitting this will make this library more approachable.

Thank you

**Comment by tschuehly** [comment]
Hey @taypo,
this isn't really needed for the library, it is only for IntelliJ that the IntelliSense works correctly.
